scripts/common.sh

```diff
@@ -202,7 +203,49 @@ function diff_values
     shift
     shift
 
-    diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    if ! diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    then
+        if [ "${DIFFVIEW:-}" == "" ]
+        then
+            printError "ERROR: values differ:"
+            printError "Expected:"
+            printError "${value1}"
+            printError "Obtained:"
+            printError "${value2}"
+        else
+            # Use user supplied diff view binary
+            printError "ERROR: values differ."
+            "$DIFFVIEW" "${file1}" "${file2}"
+        fi
+        return 1
+    fi
+}
+
+function diff_files
+{
+    (( $# >= 2 )) || fail "diff_files requires at least 2 arguments."
+
+    local file1="$1"
+    local file2="$2"
+    shift
+    shift
```

Comment:
You can discard both arguments at once since it is guaranteed that there are at least two arguments.
```suggestion
    shift 2
```

Reply:
I just removed these shifts, because the other functions are not really following this similar scheme. I think the shifts where not really needed here, right?

Reply:
Yes, for the `diff_files` I think it is fine to remove, since you don't use other arguments in the function. For example, in the `diff_values` you need the `shift` otherwise the underline `diff` could receive a wrong set of arguments.